raindrops/test_raindrops.py

```diff
@@ -0,0 +1,36 @@
+from raindrops import raindrops
+
+
+
+def test_factor_3():
```

Comment:
Let's discuss these tests in person. They look good, but some thoughts around naming that are too long to write.

Reply:
Such as `test_number_with_factor_3`

`test_number_with_factor_3_and_5`

`test_number_with_no_matching_factors`

Reply:
Also try to group the tests...which you mostly did.